vuepress/docs/next/tutorials/compose/content-types-tutorial.md

```diff
@@ -1,101 +1,70 @@
-type. Every attribute type has its own configuration.
+- `Code`: It is mandatory to enter a unique name for the Attribute key (both Attributes of a 'Composite' Attribute Type may share a `Code` name). This field supports a string value (10 characters max) of uppercase and lowercase letters, numbers and/or special characters (e.g. title).
 
-For example, to configure the **Text** Attribute Type, choose Text from
-the Types drop-down menu and then press the Add button.
+- `Name`: Enter a description of the Attribute. This field supports a string value (50 characters max) of uppercase and lowercase letters, numbers and/or special characters (e.g. Title).
 
-![image](./img/ContentTypes5.png)
+- You may have the option to declare that the Attribute is `Mandatory`, `Searchable` and/or `Can be used as a filter in lists` via toggle buttons.
 
--   You should specify the following fields
+- You may have the option to choose an Attribute `Role`. Be sure to click the `Add` button after making your selection.  
 
--   **Type:** the value is pre-filled. It represents the selected
-    attribute type.
+- The remaining fields are not mandatory and may be left empty.
 
--   **Code**: You should insert the name of the attribute. (You
-    can insert max 10 characters uppercase and lowercase letters,
-    numbers, and special characters \_ ). (e.g. title)
+:::tip
+Certain Attribute Types allow you to assign the `Role` of `jacms:title - The main title of a Content`. This informs other plugins or services that the Attribute is a title, regardless of its key, or `Code`. Entering "title" for the `Code` avoids confusion when this `Role` is selected.
+:::
 
--   **Name**: You should insert the description of the attribute.
-    (You can insert max 50 characters uppercase and lowercase letters,
-    including numbers and special characters). (e.g. Title).
+**2. Click the `Continue` button.**
 
--   You should declare if the attribute is **mandatory**,
-    **searchable**, and **to be used as a filter in the list.**
+This adds the configured Attribute to the Attribute Type.
 
--   The **Roles** section it is possible to assign the *jacms:title
-    Attribute Role* to this attribute; in this way other plugins or
-    services will know that within the CNG entity this attribute is a
-    title, whatever the key is; however, for sake of clarity, the Code
-    of the attribute is Title.
+![modifyContentType](./img/content_types4.png)
 
--   The remaining fields related to **Sections** and **Validation -
-    OGNL** sections are not mandatory and can be left empty.
+- (Optional) Complete the additional configuration steps required by your Attribute Type, if applicable (the 'Text' Attribute Type requires no additional configuration). 
 
--   Press the **Continue** button. The Text attribute type requires no
-    additional configuration step.
+- (Optional) Add other Attribute Types to your Attribute, subject to the above configuration process.
```

Comment:
does this mean an attribute type can be added to the configuration of another attribute type?

Reply:
mistake! should be add other Attribute Types to your Content Type.